.github/workflows/test-milo.yml

```diff
@@ -2,6 +2,12 @@ name: Verify Milo Changes
 
 on:
   workflow_dispatch:
+    inputs:
+      testCodeBranch:
+        description: 'Test Code Branch'
+        required: true
```

Comment:
Does this input need to be supplied by https://github.com/adobecom/milo/blob/main/.github/workflows/dispatch.yml?

Reply:
Examples from GitHub have `required: true` with a default value. I think it will use the default value if it is not supplied.